src/moj/components/multi-file-upload/multi-file-upload.js

```diff
@@ -115,7 +115,7 @@ if(MOJFrontend.dragAndDropSupported() && MOJFrontend.formDataSupported() && MOJF
   MOJFrontend.MultiFileUpload.prototype.uploadFile = function(file) {
     var formData = new FormData();
     formData.append('documents', file);
-    var item = $(this.getFileRowHtml(formData.get('documents')));
+    var item = $(this.getFileRowHtml(file));
```

Comment:
If this works, do we need formData defined at all?

Reply:
I dont think so - formData is passed to the ajax